index.js

```diff
@@ -73,26 +73,15 @@ module.exports = {
       console.log(`** Adding next.config.js with target set to 'serverless' **`)
     }
   },
-  async onBuild({ constants }) {
+  async onBuild({ constants: { PUBLISH_DIR, FUNCTIONS_SRC = DEFAULT_FUNCTIONS_SRC } }) {
     console.log(`** Running Next on Netlify package **`)
     nextOnNetlify()
 
     // Next-on-netlify puts its files into out_functions and out_publish
     // Copy files from next-on-netlify's output to the right functions/publish dirs
-
-    // TO-DO: use FUNCTIONS_DIST when internal bug is fixed
-    const { PUBLISH_DIR } = constants
-    // if (!(await pathExists(FUNCTIONS_DIST))) {
-    //   await makeDir(FUNCTIONS_DIST)
-    // }
-    const hasPublishDir = await pathExists(PUBLISH_DIR)
-    if (!hasPublishDir) {
-      await makeDir(PUBLISH_DIR)
```

Comment:
`make-dir` creates the directory if it does not exist.

Reply:
oh so it does nothing if it already exists? cool